app/assets/js/components/BatchEdit/ModalRemove.js

```diff
@@ -48,9 +48,7 @@ export default function BatchEditAboutModalRemove({
       : [];
 
   useEffect(() => {
-    if (items.length > 0) {
-      setCandidateList(setupCandidateList(items));
-    }
+    setCandidateList(items.length > 0 ? setupCandidateList(items) : []);
```

Comment:
In order to make this bulletproof from the other end, would it make more sense just to

```
setCandidateList(setupCandidateList(items));
```

and then have the first line of `setupCandidateList` be

```
if (items.length == 0) return [];
```

Reply:
Excellent, even more solid.